contracts/Pool.sol

```diff
@@ -56,11 +59,11 @@ contract Pool {
         // Block number at which the funding period has started
         uint64 startBlock;
         // The amount available when the funding period has started
-        uint192 startBalance;
+        uint128 startBalance;
```

Comment:
Does it make sense to reorder the fields to align the slot boundaries again?

Reply:
I've been considering it, but have decided not to touch it. We can't reduce the number of used slots to 1. Alignment is maintained by Solidity, it packs the fields one after another, but never cuts any in half and zero-pads the necessary gaps. I like this order too, because it's possible to touch only one slot when topping up.

Reply:
But what we can do is cramming `weightSum` and `weightCount` into the first slot. This way only updates of `amtPerBlock` will need to touch the second slot.

Reply:
Fixed